packages/rspeedy/plugin-react/src/index.ts

```diff
@@ -19,4 +19,14 @@ export type {
   ShakeVisitorConfig,
 } from '@lynx-js/react-transform'
 
+// We only export types here
+// It is encouraged to use `api.useExposed(Symbol.for('LynxTemplatePlugin'))`
+// to access the actual API
+export type {
+  LynxTemplatePlugin,
+  LynxTemplatePluginOptions,
+  TemplateHooks,
+  EncodeOptions,
```

Comment:
The only concern here is that, every breaing change in `@lynx-js/template-webpack-plugin` would now become a breaking change of `@lynx-js/react-rsbulid-plugin`.

What do you think about not exporting these types, instead install `@lynx-js/template-webpack-plugin` as a `devDependencies` in the other plugins.